### DISCUSSION TITLE
OCTANE: Calculate total request load time

### DISCUSSION THREAD

**Discussion by gunzino** [laravel/framework#48022] [Q&A]
Hi guys, has anyone figured out how to calculate request total loading time when app is deployed using Octane? I've used LARAVEL_START global variable before which is not available in Octane.

Thanks for help!

**Comment by k1rana** [comment]
+1

**Comment by ghost** [comment]
Octane keeps the app bootstrapped in memory. You could put something in the controller (not on construct) in the function that is referenced from routes to measure the execution time. Logs from the queue worker started in daemon mode work in the same way.
